### PR TITLE
LRQA-77965 | master

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LRQA-77965
Submit for master only.

cc/ @JoyceWang08211, @gaborlovasdotcom

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 1]
```diff
@@ -32,6 +32,82 @@ definition {
...
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name",
+				widgetName = "Asset Publisher");
+		}
+
+		task ("When: Configure the "SCOPE" to Global and enable "Comments" in Assert Publisher Configuration Settings") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:54]: Please change to "And" clause.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 2]
```diff
@@ -32,6 +32,82 @@ definition {
...
+			AssetPublisherPortlet.configureDisplaySettings(displaySettings = "Comments");
+
+			AssetPublisherPortlet.configureDynamicAssetSelectionPG(portletName = "Asset Publisher");
+
+			AssetPublisherPortlet.configureScope(globalSite = "Global");
+
+			IFrame.closeFrame();
+		}
+
+		task ("Then: The WC created before is displayed in the Assert Publisher") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:68]: Remove this clause and move this step in the previous clause.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 3]
```diff
@@ -32,6 +32,82 @@ definition {
...
+				siteScopeName = "Global");
+
+			LAR.exportWithAssertionOnSuccess();
+
+			var larFileName = LAR.getLarFileName();
+
+			LAR.downloadLar();
+		}
+
+		task ("And: Add a virtual instance") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:88]: Please include the login step in this clause.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 4]
```diff
@@ -32,6 +32,82 @@ definition {
...
+			PortalInstances.openVirtualInstancesAdmin();
+
+			PortalInstances.addCP(
+				mailDomain = "www.able.com",
+				virtualHost = "www.able.com",
+				webId = "www.able.com");
+		}
+
+		task ("Then: Login the virtual instance and assert that the import of the LAR process is successful by unchecking DM under CONTENT") {
+			User.firstLoginPG(
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:98]: Move to the previous clause.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
LGTM, I won't forward this PR until LRQA-77969 is merged in case of conflict in LAR.macro.

**Comment by jaydawu** [comment]
Hello @JoyceWang08211,

The related [LRQA-77969](https://github.com/brianchandotcom/liferay-portal/pull/125324) has been merged, so this pull should be approved, thanks!

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**